docs/REST API/rest-api.md

```diff
@@ -7566,12 +7566,13 @@ Deletes an existing script.
 
 | Name            | Type    | In    | Description                                                                                                                   |
 | --------------- | ------- | ----- | ----------------------------------------------------------------------------------------------------------------------------- |
+| team_id         | integer | query | _Available in Fleet Premium_. The ID of the team to filter scripts. If not specified, it will filter only scripts that are available to hosts with no team. |
 | page            | integer | query | Page number of the results to fetch.                                                                                          |
 | per_page        | integer | query | Results per page.                                                                                                             |
 
 #### Example
 
-`GET /api/v1/fleet/scripts`
+`GET /api/v1/fleet/scripts?team_id=2`
```

Comment:
No need to add the team filter here since the scripts in the response have `team_id: null`
```suggestion
`GET /api/v1/fleet/scripts`
```